docs/Example-Usage.md

```diff
@@ -0,0 +1,56 @@
+# Example Usage
+
+This is an example of the most common use case of this SDK - fetching operations and approving them.
+
+## SDK Integration
+
+Follow the [SDK Integration](./SDK-Integration.md) tutorial for SDK installation.
+
+## Example Code
+
+```swift
+// PowerAuth instance needs to be configured and a user-activated instance.
+// More about PowerAuth SDK can be found here: https://github.com/wultra/powerauth-mobile-sdk
+
+
+    fun exampleUsage(appContext: Context, powerAuth: PowerAuthSDK) {
+        // Create the WultraMobileToken instance
+        val wmt = powerAuth.createWultraMobileToken(appContext, acceptLanguage = "de")
+
+        // Launch a coroutine to handle the async operation
+        CoroutineScope(Dispatchers.IO).launch {
```

Comment:
Get rid of the Coroutines - this should be a simple example. What if the customer does not use Coroutines? Then it's just an unknown code for him. keep it simple.

```kotlin
val wmt = powerAuth.createWultraMobileToken(appContext, acceptLanguage = "de")
wmt.operations.getOperations {
   ...
}
```